crates/ir/src/module.rs

```diff
@@ -84,9 +84,9 @@ impl ModuleCtx {
     }
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct FuncRef(u32);
-entity_impl!(FuncRef);
+entity_impl!(FuncRef, "%");
```

Comment:
I'm not entirely sure why `FuncRef` itself should implement `Display`, especially because this `Display` implementation won't emit anything useful.
Would you elaborate on this change a little bit more?

Reply:
fair enough! we can always load the whole signature if you prefer, instead of printing e.g. `%1` or `%22`